### PR TITLE
Update PasteHandler for MS Edge

### PR DESCRIPTION
**Pull request by omarkilani**
The latest change from IE -> EDGE broke this.

### CODE REVIEW

**Comment by omarkilani** [comment]
I signed it!

(As Remember The Milk Inc.)

File: closure/goog/events/pastehandler.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ goog.events.PasteHandler = function(element) {
 
   if (goog.userAgent.WEBKIT ||
```
Comment (concavelenz) [inline closure/goog/events/pastehandler.js]: I wonder if we can just drop the alternate branch.

Comment (omarkilani) [inline closure/goog/events/pastehandler.js]: I guess it depends what Google's minimum supported versions are?

I'd happily drop it... :)

Comment (concavelenz) [inline closure/goog/events/pastehandler.js]: ok, lets drop it.

**Comment by concavelenz** [comment]
going with this as is until we publish a browser support policy

**Pull request merged**